tools/docker-compose-minikube/minikube/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+- name: Set home binary path directory
+  set_fact:
+    home_bin: "{{ lookup('env','HOME') }}/bin"
+
+- name: Create home binary path directory
+  file:
+    path: "{{ home_bin }}"
```

Comment:
Not a huge fan of putting things in people's home directory like this. Consider putting in under a ignored / hidden directory in the root of the repo.

Reply:
OK, it makes sense. Created a new space under the repo and ignored it on git.